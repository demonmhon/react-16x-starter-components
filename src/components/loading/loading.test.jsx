import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import Loading from './index';

describe('Loadingp component', () => {
  test('it matches the snapshot', () => {
    const component = create(<Loading />);
    expect(component.toJSON()).toMatchSnapshot();
  });

  test('has the provided class name', () => {
    const className = 'the-custom-css-class';
    const component = shallow(<Loading className={className} />);
    expect(component.hasClass(className)).toBe(true);
  });
});
