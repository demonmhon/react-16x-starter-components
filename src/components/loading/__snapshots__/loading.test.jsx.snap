// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loadingp component it matches the snapshot 1`] = `
<span
  className="starter-loading starter-loading--show"
>
  <span
    className="starter-loading__mask"
  />
  <svg
    className="starter-loading__indicator"
    viewBox="0 0 50 50"
  >
    <circle
      className="starter-loading__indicator-path"
      cx="25"
      cy="25"
      fill="none"
      r="20"
      strokeWidth="5"
    />
  </svg>
</span>
`;
